rueidiscompat/command.go

```diff
@@ -4926,3 +4926,85 @@ type ModuleLoadexConfig struct {
 	Conf map[string]interface{}
 	Args []interface{}
 }
+
+type ClusterLink struct {
+	Direction           string
+	Node                string
+	CreateTime          int64
+	Events              string
+	SendBufferAllocated int64
+	SendBufferUsed      int64
+}
+
+// ClusterLinksCmd represents the response structure for ClusterLinks.
+type ClusterLinksCmd struct {
+	val []ClusterLink
+	err error
+}
+
+func (c *ClusterLinksCmd) SetErr(err error) {
+	c.err = err
+}
+
+func (c *ClusterLinksCmd) Err() error {
+	return c.err
+}
+
+func (cmd *ClusterLinksCmd) from(res rueidis.RedisResult) {
+	arr, err := res.ToArray()
+	if err != nil {
+		cmd.SetErr(err)
+		return
+	}
+
+	val := make([]ClusterLink, 0, len(arr))
+	for _, v := range arr {
+		dict, err := v.ToMap()
```

Comment:
```suggestion
		dict, err := v.AsMap()
```
We now prefer `AsMap` over `ToMap` because `AsMap` can work with RESP2 as well.